packages/truffle/test/scenarios/commandRunner.js

```diff
@@ -2,31 +2,80 @@ const { exec } = require("child_process");
 const { EOL } = require("os");
 const path = require("path");
 
+/**
+ * Log an annotated message to stdout. This is useful to tap into a chile process'
```

Comment:
doesn't matter much but there is a typo - come to think of it, I kind of like chile better than child